src/clm/commands/add_carbon.py

```diff
@@ -34,12 +35,10 @@ def add_carbon(input_file, output_file):
         os.remove(output_file)
 
     # open buffer
-    f = open(output_file, "a+")
+    # f = open(output_file, "a+")
     # write header
     row = "\t".join(["input_smiles", "mutated_smiles", "mass", "formula", "inchikey"])
-    _ = f.write(row + "\n")
-    f.flush()
-
+    write_to_csv_file(output_file, row + "\n", "a+")
```

Comment:
I think you want to pass either a DataFrame or an iterable, so in this case an iterable with a single element, the entire text on the line (along with the newline).

Reply:
Also, write mode here I think.